jumanji/types.py

```diff
@@ -72,8 +72,9 @@ class TimeStep(Generic[Observation]):
             also valid in place of a scalar array.
         extras: environment metrics or things to be seen by the agent but not directly
```

Comment:
This becomes slightly confusing with the end of the paragraph. Maybe I would change to something along the lines: "extras: environment metrics or custom data returned by the environment not

Reply:
Have updated the description 👍